pulp_container/app/migrations/0019_DATA_distribution_model_swap.py

```diff
@@ -0,0 +1,89 @@
+# Generated by Django 2.2.19 on 2021-03-19 17:42
```

Comment:
The only thing i found was that this comment gives wrong credit. Should we remove that?